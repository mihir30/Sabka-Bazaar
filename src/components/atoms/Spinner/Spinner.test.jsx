import React from 'react';
import ReactDom from 'react-dom';
import renderer from 'react-test-renderer';
import WithSpinner from './Spinner';

describe("Spinner component", () => {
    it('renders the component properly', () => {
        const divEl = document.createElement('div');
        ReactDom.render(<WithSpinner />, divEl);
    });

    it('should have the right Spinner component snapshot', () => {
        const tree = renderer.create(<WithSpinner />).toJSON();
        expect(tree).toMatchSnapshot();
    });

});