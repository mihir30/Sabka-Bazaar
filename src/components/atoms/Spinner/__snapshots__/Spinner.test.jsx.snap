// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component should have the right Spinner component snapshot 1`] = `
<div
  className="spinner-overlay"
>
  <div
    className="spinner-container"
  />
</div>
`;
